playbook.yml

```diff
@@ -0,0 +1,80 @@
+---
+- hosts: es_0
+  roles:
+    - role: debian-common
+      prebootstrap: true
+      become: true
+    - role: debian-common
+      bootstrap: true
+      add_remove_keys: true
+      vars:
+        ports: ["22", "9300", "9200"]
+    - role: elastic
+      vars:
+        es_config:
+          node.master: true
+          node.data: false
+
+- hosts: es_1
+  roles:
+    - role: debian-common
+      prebootstrap: true
+      become: true
+    - role: debian-common
+      bootstrap: true
+      add_remove_keys: true
+      vars:
+        ports: ["22", "9300", "9200"]
+    - role: elastic
+      vars:
+        es_config:
+          node.master: false
+          node.data: true
+
+- hosts: es_2
+  roles:
+    - role: debian-common
+      prebootstrap: true
+      become: true
+    - role: debian-common
+      bootstrap: true
+      add_remove_keys: true
+      vars:
+        ports: ["22", "9300", "9200"]
+    - role: elastic
+      vars:
+        es_config:
+          node.master: false
+          node.data: true
+
+- hosts: grafana
+  roles:
+    - role: debian-common
+      prebootstrap: true
+      become: true
+    - role: debian-common
+      bootstrap: true
+      add_remove_keys: true
+      vars:
+        ports: ["22", "3000"]
+    - role: grafana
+
+- hosts: ralph
+  roles:
+    - role: debian-common
+      prebootstrap: true
+      become: true
+    - role: debian-common
+      bootstrap: true
+      add_remove_keys: true
+      vars:
+        ports: ["22"]
+    - role: pyenv
+      vars:
+        pyenv_owner: admin
+        pyenv_virtualenvs:
+          - venv_name: ralph_venv
+            py_version: 3.9.15
```

Comment:
Declarative, host/role related configuration variables should live in the inventory: users are not supposed to modify a playbook to achieve such tasks (see `ports` or `py_version` for example).